docs/reference/module-options.md

```diff
@@ -136,4 +138,257 @@ You might use that value as a fallback for user-editable fields.
 </h2>
 ```
 
-## Accessing module options
\ No newline at end of file
+## Options for all doc type modules
+
+Option settings in this section apply to all modules that extend `@apostrophecms/doc-type` ([doc type](glossary.md#doc) modules). These include all piece and page types.
+
+- [`adminOnly`](#adminonly)
```

Comment:
Only users with the sitewide admin permission can read or write docs of this type. `@apostrophecms/user` itself is a good example.

Reply:
description updated below